tests/v2/validation/provisioning/rke2/provisioning_node_driver_test.go

```diff
@@ -142,7 +147,7 @@ func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioningRKE2Cluster() {
 				for _, cni := range r.cnis {
 					name += " cni: " + cni
```

Comment:
I'm not opposed to it. I didn't put it here initially for regex purposes and since it may get convoluted if PSA is disabled in the user configuration.